src/sandbox.rkt

```diff
@@ -334,16 +327,21 @@
      (define start-test-score (errors-score start-test-errs))
      (define start-cost (expr-cost start-expr repr))
 
-     ;; From all the targets, pick the lowest cost target
-     (define target
-      ; If the list is empty, return false
-      (if (empty? targets)
-        #f
-        (argmin (lambda (target) (alt-cost (alt-analysis-alt target) repr)) targets)))
+     (define target-cost-score
+      (for/list ([target targets])
+        (define target-expr (alt-expr (alt-analysis-alt target)))
+        (define tar-cost (expr-cost target-expr repr))
+        (define tar-score (errors-score (alt-analysis-test-errors target)))
+
+        (list tar-cost tar-score)))
```

Comment:
Intentation is weird but whatever.

Reply:
Sorry about that. I'll fix this and the `best-score` a few lines down.